components/accept-financial-contributions/ApplyToHost.js

```diff
@@ -0,0 +1,194 @@
+      },
+      infoParagraph: {
+        id: 'fiscalHost.apply.info',
+        defaultMessage:
+          "With this option, you don't need to set up a legal entity and bank account for your project. The fiscal host will hold funds on your behalf, and take care of accounting, invoices, tax, admin, payments, and liability. Most hosts charge a fee for this service (you can review these details on the host's page before confirming).",
+      },
+    });
+  }
+
+  render() {
+    const { data, intl } = this.props;
+
+    if (!data.allHosts || !data.allHosts.collectives) {
+      return <div />;
+    }
+
+    const collectives = [...data.allHosts.collectives];
+
+    if (collectives.length === 0) {
+      return <div />;
+    }
+
+    return (
+      <Fragment>
+        <Box mb={4} mt={5}>
+          <H1 fontSize={['H5', 'H3']} lineHeight={['H5', 'H3']} fontWeight="bold" color="black.900" textAlign="center">
+            {intl.formatMessage(this.messages.header)}
+          </H1>
+        </Box>
+        <Container alignItems="flex-end" display="flex" pl={7} width={1}>
+          <img src="/static/images/create-collective/umbrellaPot.png" width="264px" height="184px" />
+          <Box maxWidth={'400px'}>
+            <H2 fontSize="H5" color="black.900">
+              {intl.formatMessage(this.messages.applyToHost)}
+            </H2>
+            <P mb={3} color="black.700">
+              {intl.formatMessage(this.messages.infoParagraph)}
+            </P>
+          </Box>
+        </Container>
+        <Container alignItems="flex-start" display="flex" px={6} py={[20, 60]} width={1}>
+          <AllCardsContainer>
+            {collectives.map(collective => (
+              <CollectiveCardContainer key={collective.id}>
+                <HostCollectiveCard
+                  key={collective.id}
+                  hostCollective={collective}
+                  applyingCollective={this.props.collective}
+                />
+              </CollectiveCardContainer>
+            ))}
+          </AllCardsContainer>
+          <InterestedContainer
+            display="flex"
+            flexDirection="column"
+            alignItems="center"
+            minHeight="750px"
+            px={[2]}
+            pt={6}
+            mt={5}
+          >
+            <H3 fontSize="H5" fontWeight="500" color="black.600" textAlign="center">
+              {intl.formatMessage(this.messages.interestedInHosting)}
+            </H3>
+            <img src="/static/images/create-collective/climateIllustration.png" width="192px" height="192px" />
+            <ExternalLink href={FISCAL_HOST_LINK} openInNewTab>
+              <StyledButton buttonStyle="dark" mt={[2, 3]} mb={3} px={3}>
+                {intl.formatMessage(this.messages.becomeHost)}
+              </StyledButton>
+            </ExternalLink>
+          </InterestedContainer>
+        </Container>
+      </Fragment>
+    );
+  }
+}
+
+const getHostsQuery = gqlV2`
```

Comment:
Maybe it's because it's WIP but it says gqlV2 while being a GraphQL v1 query.

Reply:
Yep I had not realised that I had more changing to do. That's all taken care of now.